### PR TITLE
Spotlight for CNCF DHHWG

### PR DESCRIPTION
**Pull request by sandeepkanabar**
This is a spotlight for the CNCF Deaf and hard-of-hearing working group (DHHWG)

FIxes: #529

### CODE REVIEW

**Comment by sftim** [comment]
/hold

Let's check whether this actually belongs on a different blog (eg: CNCF blog).

**Comment by sandeepkanabar** [comment]
> /hold
> 
> Let's check whether this actually belongs on a different blog (eg: CNCF blog).
Understand Tim. The aim here is to drive inclusion among contributors and kubernetes folks so that they can be allies to the DHHWG and can help mentor folks from the WG. The CNCF will also publish a blog on the WG this Deaf Awareness Month, in addition to a host of other activities too. While there is an audience overlap, publishing on both blogs will allow us to reach a much bigger audience.

**Comment by sftim** [comment]
Maybe we get the CNCF to publish an article and at the same moment we talk about signing in American Sign Language (or maybe other languages too?) and what's available for contributors who use this.

**Comment by sandeepkanabar** [comment]
> Maybe we get the CNCF to publish an article and at the same moment we talk about signing in American Sign Language (or maybe other languages too?) and what's available for contributors who use this.

Hi Tim - As mentioned other [thread](https://github.com/kubernetes/website/pull/48021#issuecomment-2365262700), CNCF would be bringing out a blog of its own and also holding a bunch of other activities too (including Sign Language).

**Comment by sftim** [comment]
Let's gather evidence, and possibly revise the article, so that we're confident that this is right for this blog.

**Comment by sandeepkanabar** [comment]
> Let's gather evidence, and possibly revise the article, so that we're confident that this is right for this blog.

Sure Tim. Would you mind clarifying to whom do we need to reach out to gather evidence and find a way forward for this?

**Comment by sftim** [comment]
@sandeepkanabar I think the task here is to work with SIG ContribEx Comms to engage with the CNCF and understand how to do a spotlight article that is about a CNCF WG (or TAG) rather than a K8s WG or SIG.

We're actually defining a _pattern_ here so ideally we make sure we don't get it too wrong; otherwise, we end up perpetuating some of the snags.

---

Things to watch out for:
- “sign language“ isn't a language, there are lots of sign languages. People who know one sign language may not know any other.
  - I know the interview didn't cover this, but maybe a preface could help?
  - If we're willing to revise the spotlight discussion after the fact, that's likely to be more inclusive
- Deaf Awareness Month is for America and selected other countries, AIUI, not the world (it's still OK to align to it)
- use **bold** rather than CAPITALS for emphasis
- if we can explain what a CNCF is and why we're working with the CNCF, that's good too. Maybe in the article introduction?

Can we explain what _Kubernetes_ is doing around accessibility and hearing? That would be nice to mention alongside this interview.

**Comment by kaslin** [comment]
Hey, @sftim, I think an important detail has been missed here. @sandeepkanabar is a member of the Kubernetes SIG ContribEx Comms team who is also Deaf and a member of the DHHWG himself. He wanted to write this article to highlight the work that group has been doing. So this blog post and Sandeep's work with us is actually an example of what Kubernetes is doing around accessibility. I had originally suggested that Sandeep include a section here about his specific experience contributing to Kubernetes and how the DHHWG has helped him. @sandeepkanabar maybe you could add a little bit about who you are at the beginning of the article and then maybe a closing paragraph about how being part of the DHHWG has helped you become a Kubernetes contributor?

**Comment by sandeepkanabar** [comment]
> Things to watch out for:
> 
> * “sign language“ isn't a language, there are lots of sign languages. People who know one sign language may not know any other.
>   
>   * I know the interview didn't cover this, but maybe a preface could help?
>   * If we're willing to revise the spotlight discussion after the fact, that's likely to be more inclusive

I appreciate you striving to be inclusive, @sftim. But the deaf world has a `spectrum`. Some sign, some don't, some lip-read, some rely on captions and many more. As a `deaf` person myself, my friends in the DHHWG are pretty diverse and there are American/British/Indian/International/Vietname/<other> sign language users. It is to **include all of them** that I intentionally used the umbrella term `Sign Language` so that no one feels left out. If you know of a better and more inclusive term, please suggest and I'll be happy to incorporate.

**Comment by sandeepkanabar** [comment]
> @sandeepkanabar maybe you could add a little bit about who you are at the beginning of the article and then maybe a closing paragraph about how being part of the DHHWG has helped you become a Kubernetes contributor?

That's an excellent suggestion, @kaslin . Let me think it through and add that context to the blog.

**Comment by fsmunoz** [comment]
(adding my comments that I added to the `website` PR by mistake, to help concentrate the discussion here)

I agree that this is a corner-case, since it's a CNCF Working Group. The reasons why I think that it fits within k8s.dev (at least), and to add to the discussion, re:

* The link to the Kubernetes ecosystem is the starting point of the conversation, and explicitly mentioned and discussed.
* This WG has presented in KubeCons (that's the first time I heard them), and the subject matter is one which doesn't fit a k8s WG, while at the same time being important to k8s.
* It specifically addresses how to make KubeCon itself more accessible.

This doesn't mean it wouldn't also be suitable for the CNCF blog (although I'm not knowledgeable about the publication process or criteria there), just that I personally see it with sufficient contact area with the Kubernetes community. I do however understand that these might be seen as insufficient for others.

**Comment by sftim** [comment]
Making this about how that WG is helping people contribute to Kubernetes? It takes away all the concerns I had. Let's do it.

**Comment by sftim** [comment]
I think the CNCF could use a contributor blog similar to how Kubernetes has that split. But, I don't have the capacity to move that conversation forward.

File: content/en/blog/2024/cncf-dhhwg/cncf-dhhwg-spotlight.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+**SK: Speaking of history, since its launch, how has the DHHWG evolved? Could you highlight some of the key milestones or achievements the group has reached recently?**
+
+**CP**: Our WG is about a year old. It started with a few deaf engineers and me brainstorming how to make KubeCon more accessible. We published an initial draft of [Best practices for an inclusive conference](https://contribute.cncf.io/accessibility/deaf-and-hard-of-hearing/conference-best-practices/) and shared it with the LF events team. KubeCon Chicago was two months later, and we had a couple of deaf attendees. It was the FIRST KubeCon accessible to deaf signers. [Destiny](https://www.linkedin.com/in/destiny-o-connor-28b2a5255/), one of our co-chairs, even participated in a [keynote panel](https://youtu.be/3WJ_s4Jvbsk?si=iscthTiCyMxoMUqY&t=347). It was incredible how quickly everything happened!
```
Comment (sftim) [inline content/en/blog/2024/cncf-dhhwg/cncf-dhhwg-spotlight.md:27]:
```suggestion
**CP**: Our WG is about a year old. It started with a few deaf engineers and me brainstorming how to make KubeCon more accessible. We published an initial draft of [Best practices for an inclusive conference](https://contribute.cncf.io/accessibility/deaf-and-hard-of-hearing/conference-best-practices/) and shared it with the LF events team. KubeCon Chicago was two months later, and we had a couple of deaf attendees. It was the **first** KubeCon accessible to deaf signers. [Destiny](https://www.linkedin.com/in/destiny-o-connor-28b2a5255/), one of our co-chairs, even participated in a [keynote panel](https://youtu.be/3WJ_s4Jvbsk?si=iscthTiCyMxoMUqY&t=347). It was incredible how quickly everything happened!
```

File: content/en/blog/2024/cncf-dhhwg/cncf-dhhwg-spotlight.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+**SK: Are there any misconceptions about the deaf and hard-of-hearing community in tech that you'd like to address?**
+
+**CP**: Deaf and hard of hearing individuals are very diverse — there is no one-size-fits-all. Some deaf people are oral (speak), others sign, while some lip read or prefer captions. It generally depends on how people grew up. While some people come from deaf families and sign language is their native language, others were born into hearing families who may or may not have learned how to sign. Some deaf people grew up surrounded by hearing people, while others grew up deeply embedded in Deaf culture. Hard-of-hearing individuals, on the other hand, typically can communicate well with hearing peers one-on-one in quiet settings, but loud environments or conversations with multiple people can make it hard to follow the conversation. Most rely heavily on captions. Each background and experience will shape their communication style and preferences. In short, what works for one person, doesn't necessarily work for others. So NEVER ASSUME and ALWAYS ASK about accessibility needs and preferences.
```
Comment (sftim) [inline content/en/blog/2024/cncf-dhhwg/cncf-dhhwg-spotlight.md:40]:
```suggestion
**CP**: Deaf and hard of hearing individuals are very diverse — there is no one-size-fits-all. Some deaf people are oral (speak), others sign, while some lip read or prefer captions. It generally depends on how people grew up. While some people come from deaf families and sign language is their native language, others were born into hearing families who may or may not have learned how to sign. Some deaf people grew up surrounded by hearing people, while others grew up deeply embedded in Deaf culture. Hard-of-hearing individuals, on the other hand, typically can communicate well with hearing peers one-on-one in quiet settings, but loud environments or conversations with multiple people can make it hard to follow the conversation. Most rely heavily on captions. Each background and experience will shape their communication style and preferences. In short, what works for one person, doesn't necessarily work for others. So **never assume** and **always ask** about accessibility needs and preferences.
```

**Review by sftim — COMMENTED** [review]
A couple of formatting tweaks.

**Comment by sandeepkanabar** [comment]
> Making this about how that WG is helping people contribute to Kubernetes? It takes away all the concerns I had. Let's do it.



> A couple of formatting tweaks.

Thanks so much for catching this @sftim . Incorporated the suggestions.

**Comment by sandeepkanabar** [comment]
> Making this about how that WG is helping people contribute to Kubernetes? It takes away all the concerns I had. Let's do it.

Thank you @sftim and @kaslin. Would you mind re-reviewing the article please? Have modified the intro and also the conclusion as @kaslin suggested. 

Also, ensured to incorporate the same changes in the [article for website](https://github.com/kubernetes/website/pull/48021) as well

**Comment by sandeepkanabar** [comment]
> How about the 30th as publication date (both articles)? Let's try for that.

Thanks so much @sftim. That's still _Deaf Awareness Month_, so that'd be great. Thank you!

**Comment by sandeepkanabar** [comment]
@sftim - changed the published date to 30th in the header.

**Comment by sandeepkanabar** [comment]
> /lgtm /approve

Thank you for all your support @sftim . V much appreciate you going above and beyond for this.
